fontforge/sfd.c

```diff
@@ -3583,6 +3633,70 @@ static void rle2image(struct enc85 *dec,int rlelen,struct _GImage *base) {
     }
 }
 
+enum MIME { PNG }; // We only understand PNG for now.
+
+enum MIME SFDGetImageMIME(FILE *sfd) {
```

Comment:
Should probably be named `SFDGetImageXMIME` or (if the tag is renamed to `Image2` on @khaledhosny's suggestion,  `SFDGetImage2MIME`. `SFDGetImagePNG` and the others can stay as is.